packages/app/cypress.config.ts

```diff
@@ -11,19 +11,15 @@ export default defineConfig({
     runMode: 2,
     openMode: 0,
   },
-
-  testFiles: '**/*.{spec,cy}.{js,ts,tsx,jsx}',
+  nodeVersion: 'system',
```

Comment:
This should be removed here and in other configs that this got added to.

Reply:
https://github.com/cypress-io/cypress/pull/19319/commits/9a13720f8d4c881a8edac8408c9239a353fc8ac5